### DISCUSSION TITLE
Top products scope

### DISCUSSION THREAD

**Discussion by dauom** [geeksblabla/blanat#24] [Q&A] [answered]
do we print the top 5 cheapest products in the cheapest city, or the top 5 cheapest overall in all cities?

**Comment by chelzakaria** [comment]
~~Top 5 cheapest products in the cheapest city~~
 
Update : Top 5 cheapest overall in all cities (Based on the example given)

**Reply by aboullaite** [reply, marked as answer]
Top 5 cheapest products in the cheapest city
